ev3dev/core.py

```diff
@@ -605,9 +616,20 @@ def time_sp(self):
     @time_sp.setter
     def time_sp(self, value):
         self.set_attr_int('time_sp', value)
+# ~autogen
 
+    def wait_for_running(self):
+        while True:
+            if 'running' in self.state:
+                return
+            sleep(0.01)
```

Comment:
You beat me to that comment @ddemidov - it's a helper function, you want the timeout with default, not a fixed counter. And specifying the sleep interval is a nice touch.

For consistency with other langauges that may provide their own helpers, please specify the `timeout` and `interval` in milliseconds and have the function translate to the language specific units.

I don't think there's any benefit to higher than msec resolution.

You might get some pushback on the use of `lambda` to create the anonymous methods `started` and `stopped` but I think it's elegant.